libbroker/broker/internal/metric_factory.cc

```diff
@@ -6,144 +6,137 @@ namespace broker::internal {
 
 // -- 'imports' to safe ourselves some typing ----------------------------------
 
-using dbl_gauge = metric_factory::dbl_gauge;
```

Comment:
Before, we had two distinct types per metric: integers and floating points. `prometheus-cpp` only has only type for each metric, so I had to remove the old aliases and add a new one. It's not my mental model to see this is as a renaming, it's a replacement.

I guess it comes down to how you look at it, I believe you mean I could've kept the `dbl_` prefix at first and then drop that later? To me that would make semantically no sense, because having the `dbl_` prefix implies that there is more than one option, so I hadn't considered it. But I agree that it might make the commit history a bit nicer to do it that way.